### PR TITLE
Add nav btns functionality

### PR DESCRIPTION
**Pull request by foxwellm**

### CODE REVIEW

File: src/Components/Navbar.js [hunk 1]
```diff
@@ -6,18 +6,21 @@ export default class Navbar extends Component {
     super();
   }
 
- 
+  toggleAdvancedSearch = () => {
+    document.querySelector('.AdvancedSearch').classList.toggle('AdvancedSearchClicked');
+  }
+
```
Comment (foxwellm) [inline src/Components/Navbar.js:12]: now AdvancedSearch is brought out through css class

File: src/Components/Navbar.js [hunk 2]
```diff
@@ -6,18 +6,21 @@ export default class Navbar extends Component {
...
   render() {
     return (
       <nav className="Navbar">
         <div class="filter-buttons">
           <h2 class="title">GAMEFLIX</h2>
           <div class="filters-column-one">
-            <p class="reset-games" onClick={() => this.props.resetAllGames('')}>All Games</p>
-            <p class="filter-boardgames" onClick={() => this.props.resetAllGames(6)}>Board Games</p>
+            <p class="reset-games" data-nav="resetall" onClick={this.props.resetAllGames}>All Games</p>
```
Comment (JustinD85) [inline src/Components/Navbar.js:19]: ohhh nice use of dataset!

**Review by JustinD85 — APPROVED** [review]

File: src/Components/Navbar.js (hunk 1 above)
Comment (whitneyfirth) [reply src/Components/Navbar.js:12]: Nice thinking!

File: src/Components/App.js [hunk 3]
```diff
@@ -93,29 +96,30 @@ class App extends Component {
...
+        document.querySelector('#board').checked = true;
+        filteredGames = games.filter(game => game.genre_ID.includes(6))
+        searchPageCheck = true;
+        break;
+      case 'resetcard':
+        filterOptions.type.card = true;
+        document.querySelector('#card').checked = true;
+        filteredGames = games.filter(game => game.genre_ID.includes(7))
+        searchPageCheck = true;
+        break;
```
Comment (whitneyfirth) [inline src/Components/App.js:122]: Nice use of the data attributes, this makes it easily readable!

**Review by whitneyfirth — APPROVED** [review]

**Pull request merged**